arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -105,6 +105,14 @@
 	pinctrl-0 = <&pinctrl_rgmii0_default &pinctrl_rgmii0_driving>;
 };
 
+&mac1 {
```

Comment:
Has the NCSI tested yet on any of these platforms?

Reply:
yes tested on Congo.